template.yaml

```diff
@@ -83,8 +83,8 @@ Resources:
       LifecycleConfiguration:
         Rules:
           - Id: "ExpiryPolicy"
-            ExpirationInDays: 1
-            NoncurrentVersionExpirationInDays: 1
+            ExpirationInDays: 14
+            NoncurrentVersionExpirationInDays: 14
```

Comment:
Not sure if need `NoncurrentVersionExpirationInDays` at all. Do we use versioning? Isn't every event in bucket unique?

Reply:
It also seems like `NoncurrentVersionExpirationInDays` is deprecated.

Reply:
Try to remove it and see what happens?